include/swift/ClangImporter/ClangImporter.h

```diff
@@ -705,6 +705,9 @@ bool isCxxConstReferenceType(const clang::Type *type);
 /// Determine whether this typedef is a CF type.
 bool isCFTypeDecl(const clang::TypedefNameDecl *Decl);
 
+// Determine whether type is a c++ foreign reference type.
+bool isForeignReferenceTypeWithoutImmortalAttrs(const clang::QualType type);
```

Comment:
Minor: no need for `const` here.

Reply:
You mean only in the declaration in the ClangImporter.h file or also in the definition in the ClangImporter.cpp file?
I wanted to keep the signature of `isForeignReferenceTypeWithoutImmortalAttrs` similar to `isForeignReferenceType`

Reply:
Ah I see, there are existing functions that have `const` here. That's fine then, feel free to disregard this comment 🙂